src/main/java/org/gridsuite/shortcircuit/server/repositories/ResultRepository.java

```diff
@@ -32,4 +35,9 @@ public interface ResultRepository extends JpaRepository<ShortCircuitAnalysisResu
     Optional<ShortCircuitAnalysisResultEntity> findWithFaultResultsAndFeederResultsByResultUuid(UUID resultUuid);
 
     void deleteByResultUuid(UUID resultUuid);
+
+    @Modifying
+    @Query("UPDATE ShortCircuitAnalysisResultEntity r SET r.debugFileLocation = :debugFileLocation WHERE r.resultUuid = :resultUuid")
+    int updateDebugFileLocation(@Param("resultUuid") UUID resultUuid, @Param("debugFileLocation") String debugFileLocation);
```

Comment:
Why do you need a "native" query and not doing it with JPA ? It's a simple update

Reply:
To me, we use native query when we spot a case where JPA gives bad performance and when an optimization is required. It is not the case here as we only update a single object and a single row in the database. 
A simple use of JPA makes the job

Reply:
OK! DONE in https://github.com/gridsuite/shortcircuit-server/pull/151/commits/c2bfe427479da4f1ff3dcff873036c0cb330da72